string transcription
time audio_recieved
float32 duration